src/Data/Vec/Properties.agda

```diff
@@ -76,13 +77,84 @@ unfold-take : ∀ n {m} x (xs : Vec A (n + m)) → take (suc n) (x ∷ xs) ≡ x
 unfold-take n x xs with splitAt n xs
 unfold-take n x .(xs ++ ys) | xs , ys , refl = refl
 
+take-distr-zipWith : ∀ {m n} → (f : A → B → C) →
+                     (u : Vec A (m + n)) → (v : Vec B (m + n)) →
+                     take m (zipWith f u v) ≡ zipWith f (take m u) (take m v)
+take-distr-zipWith {m = zero} f u v = refl
+take-distr-zipWith {m = suc m} f (u ∷ us) (v ∷ vs) = begin
```

Comment:
Very very minor but any chance you can align the arguments in your functions as it makes them easier to read?

Reply:
Done